weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsPage.tsx

```diff
@@ -100,14 +103,13 @@ export const CallsTable: React.FC<{
   const lowLevelFilter: CallFilter = useMemo(() => {
     return convertHighLevelFilterToLowLevelFilter(orm, effectiveFilter);
   }, [effectiveFilter, orm]);
-  const runs = useRunsWithFeedback(
-    {
-      entityName: props.entity,
-      projectName: props.project,
-      streamName: 'stream',
-    },
-    lowLevelFilter
-  );
+  const streamId = {
```

Comment:
Maybe just call this stream? I wouldn't expect something named Id to be an object.

Reply:
👍